tests/lib/nested.sh

```diff
@@ -1041,12 +1063,19 @@ nested_start_core_vm_unit() {
         # storage to
         PARAM_ASSERTIONS="-drive if=none,id=stick,format=raw,file=$NESTED_ASSETS_DIR/assertions.disk,cache=none,format=raw -device nec-usb-xhci,id=xhci -device usb-storage,bus=xhci.0,removable=true,drive=stick"
     fi
-    if nested_is_core_20_system; then
+    if nested_is_core_20_system || nested_is_core_22_system; then
         # use a bundle EFI bios by default
         PARAM_BIOS="-bios /usr/share/ovmf/OVMF.fd"
         local OVMF_CODE OVMF_VARS
         OVMF_CODE="secboot"
         OVMF_VARS="ms"
+
+        if nested_is_core_22_system; then
```

Comment:
@sergiocazzolato did you file a LP bug about the _4M vars files from jammy not working? please leave a link to that bug here